test/basic-querying.test.js

```diff
@@ -475,21 +507,26 @@ describe('basic-querying', function() {
       }}, function(err, users) {
         should.not.exist(err);
         users.should.have.property('length', 2);
-        users[0].name.should.equal('George Harrison');
+        if (!isCassandraConnector) {
+          users[0].name.should.equal('George Harrison');
+          for (var ix = 0; ix < users.length; ix++) {
+            users[ix].vip.should.be.false();
+          }
+        };
         done();
       });
     });
 
-    it('supports non-empty inq', function() {
+    bdd.itIf(!isCassandraConnector, 'supports non-empty inq', function() {
       // note there is no record with seq=100
       return User.find({where: {seq: {inq: [0, 1, 100]}}})
         .then(result => {
           const seqsFound = result.map(r => r.seq);
-          should(seqsFound).be.oneOf([0, 1], [1, 0]);
+          should(seqsFound).eql([0, 1]);
         });
     });
 
-    it('supports empty inq', function() {
+    bdd.itIf(!isCassandraConnector, 'supports empty inq', function() {
```

Comment:
For use cases like this, there should be a flag in `connectorCapabilities` object that is set by the connector and which controlls which tests are skipped.

```js
bdd.itIf(connectorCapabilities.emptyInq !== false, 'supports empty inq', function() {
  // ...
});
```

You can `grep -r connectorCapabilities test` to see other places using `connectorCapabilities` object.